symplyphysics/laws/kinematic/vector/absolute_velocity_of_arbitrary_motion.py

```diff
@@ -0,0 +1,66 @@
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    QuantityVector,
+    add_cartesian_vectors,
+    scale_vector,
+)
+
+# Description
+## Suppose two reference frames, one of which is fixed (S) and the other one is moving (S'). The movement of the
+## body relative to fixed frame S is called absolute movement. The movement of the body relative to moving frame S'
+## is called relative movement. And the movement of the body due to the movement of reference frame S' is called transfer
+## velocity. The absolute velocity is the sum of these two velocities.
+
+# Law: v_abs = v_rel + v_tr
```

Comment:
It looks very similar to https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/kinematic/vector/rolling_velocity_is_translational_plus_rotational_velocities.py
Could you generalize this law, and remove redundant law?

Reply:
I think this one is general enough to encompass that one. We can get rid of that one.

Reply:
Yes, but I think we can mention, that this law applies to rolling motion.